compliance/collection/nodeinventorizer/caching_inventorizer.go

```diff
@@ -66,17 +66,14 @@ func (c *CachingScanner) Scan(nodeName string) (*storage.NodeInventory, error) {
 	backoffDuration := c.initialBackoff
 	if cache != nil {
 		backoffDuration = c.validateBackoff(cache.BackoffDuration)
-	}
-
-	if backoffDuration > c.initialBackoff {
 		log.Warnf("Found existing node scan backoff file - last scan may have failed. Waiting %v seconds before retrying", backoffDuration.Seconds())
 		c.backoffWaitCallback(backoffDuration)
 	}
 
 	// Write backoff duration to cache
 	backoff := inventoryWrap{BackoffDuration: c.calcNextBackoff(backoffDuration)}
 	if err := writeInventoryWrap(backoff, c.inventoryCachePath); err != nil {
-		log.Warnf("Error writing node scan backoff file: %v", err)
+		return nil, errors.Wrap(err, " writing node scan backoff file")
```

Comment:
Getting rid of extra space
```suggestion
		return nil, errors.Wrap(err, "writing node scan backoff file")
```